api/utilities.py

```diff
@@ -52,21 +53,18 @@ def get_datetime_range(datetime_string: str | None) -> Tuple[Timestamp, Timestam
     return start_datetime, end_datetime
 
 
-async def get_current_parameter_names(ttl_hash=None):
+# @cached(ttl=600)
+async def get_current_parameter_names():
     """
     This function get a set of standard_names currently in the datastore
     The ttl_hash should be a value that is updated at the same frequency
     we want the lru_cache to be valid for.
     """
 
-    @lru_cache(maxsize=1)
-    async def async_helper(ttl_hash):  # pylint: disable=unused-arguement
-        unique_parameter_names = dstore.GetTSAGRequest(attrs=["parameter_name"])
-        unique_parameter_names = await getTSAGRequest(unique_parameter_names)
-
-        return set([i.combo.standard_name for i in unique_parameter_names.groups])
+    unique_parameter_names = dstore.GetTSAGRequest(attrs=["parameter_name"])
+    unique_parameter_names = await getTSAGRequest(unique_parameter_names)
 
-    return await async_helper(ttl_hash)
+    return set([i.combo.parameter_name for i in unique_parameter_names.groups])
 
 
 async def verify_parameter_names(parameter_names: list) -> None:
```

Comment:
This function assumes a list of parameters is coming in, while actually a comma separated list is passed.